src/main/java/com/google/edith/DealItem.java

```diff
@@ -27,6 +30,7 @@ public String toString() {
   private double weight;
   private String comment;
   private double unitPrice;
+  private String expiration;
```

Comment:
"expiration" could be a bit vauge, how about expirationTime? Or something similar